frontend/src/components/logged_in/NewEvent.js

```diff
@@ -12,7 +12,7 @@ import EventFields from './EventFields';
 function NewEvent() {
     const navigate = useNavigate();
 
-    const [calendars, , refreshCalendars] = useContext(CalendarContext);
+    const {calendars, refreshCalendars} = useContext(CalendarContext);
```

Comment:
I like how many params you're using in CalendarContext, very helpful and should keep coding easy.